scrape/server/flask_app/repo/users.py

```diff
@@ -76,6 +92,45 @@ def delete_user_by_email(self):
         count = cursor.rowcount
         cursor.close()
 
+    # def validate_register(self, first_name: str, last_name: str, email: str, password: str):
```

Comment:
Since we're using `validate_register` by passing user object, we don't need to comment this out; remove it for cleanliness